src/LanguageServer/ExternalAccess/Copilot/Handler/RequestContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Copilot;
+
+internal readonly struct RequestContext(LspRequestContext context)
+{
+    /// <inheritdoc cref="LspRequestContext.Solution"/>
+    internal Solution? Solution => context.Solution;
```

Comment:
since everything requires a solution, this can be:

```suggestion
    public Solution Solution => context.Solution ?? throw ExceptionUtilities.Unreachable();
```